test/testing_test.rb

```diff
@@ -166,6 +166,41 @@ def plan_input
       end
     end
 
+    describe "in thread executor hanging test" do
+      class BreakingAction < ::Dynflow::Action
+        def run(event = nil)
+          if output[:suspended].nil?
+            output[:suspended] = true
+            suspend do |action|
+              world.clock.ping(action, 1000, nil)
+            end
+          end
+        end
+      end
+
+      let :world do
+        WorldFactory.create_world(Dynflow::Testing::InThreadWorld)
+      end
+
+      let :execution_plan do
+        world.plan(BreakingAction)
+      end
+
```

Comment:
I would recommand being positive (using "it works" rather than "it breaks" attitude :)

Reply:
Updated to something a little more positive :)